sql/core/src/test/resources/sql-tests/inputs/higher-order-functions.sql

```diff
@@ -12,9 +12,14 @@ select transform(zs, z -> z) as v from nested;
 
 -- Transform an array
 select transform(ys, y -> y * y) as v from nested;
+-- use non reversed keywords
+select transform(ys, left -> left * left) as v from nested;
+
 
 -- Transform an array with index
 select transform(ys, (y, i) -> y + i) as v from nested;
+-- use non reversed keywords
+select transform(ys, (cost, i) -> cost + i) as v from nested;
```

Comment:
Can you move the two tests above into the end of the file to make diff smaller in `sq.out`.

Reply:
Also, can you add some tests for error cases if ansi=true?

Reply:
thanks for you suggestion. I just moved them to the end of this file, and add some cases to verify with ansi flag.